.gitattributes

```diff
@@ -0,0 +1,4 @@
+# Configure git to use `ansible-vault view` for diff'ing these files with:
+# git config --global diff.ansible-vault.textconv "ansible-vault view"
+etc/kayobe/secrets.yml diff=ansible-vault merge=binary
+etc/kayobe/kolla/passwords.yml diff=ansible-vault merge=binary
```

Comment:
What happens if I haven't applied the above git config?

Reply:
Wasn't 100% sure so I've just tested with git version 2.22.0. 
The git command above puts the following in ~/.gitconfig:
```
[diff "ansible-vault"]
	textconv = ansible-vault view
```
The absence of the above means the .gitattributes file is essentially ignored and `git diff alaska-alt-1 alaska-prod -- etc/kayobe/secrets.yml` for example reverts to the standard 3 way diff behaviour.